book/accessibility.md

```diff
@@ -2335,136 +2297,87 @@ should now look something like this:
+devices. The idea of [responsive design][responsive-design] means
+designing websites to work well on any kind of browser screens and
+contexts. Responsive design can be viewed as a kind of accessibility.
 
-* *Button role*: Add support for the `button` value of the `role` attribute.
+[responsive-design]: https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Responsive_Design
+
+*Threaded accessibility*: The accessibility code currently speaks text
+on the browser thread, and blocks the browser thread while it speaks.
+This is pretty annoying. Solve this by moving the speaking to a new
+accessibility thread that is in charge of speaking the document.
 
-* *High-contrast mode*: Implement high-contrast [forced-colors] mode. As part
-of this, draw a rectangular *backplate* behind all lines of text in order to
-ensure that there is sufficient contrast (as [defined][contrast] by the WCAG
-specification) between  foreground and background colors. Also check the
-contrast of the default style sheets I provided in this chapter---do they meet
-the requirements?
+*High-contrast mode*: Implement high-contrast [forced-colors] mode.
+This should replace all colors with one of a small set of
+[high-contrast][contrast] colors. You should also draw a rectangular
+*backplate* behind all lines of text in order to ensure that there is
+sufficient contrast between the text and whatever is behind it.
 
 [forced-colors]: https://developer.mozilla.org/en-US/docs/Web/CSS/@media/forced-colors
 
-* *Custom inputs*: Implement an `<input>` element with a `<div>` completely in
-JavaScript. Make sure that it's represented correctly in the accessibility tree
-and participates correctly in form submission.
-
-* *Threaded accessibility*: The accessibility code currently speaks on the
-main thread, which creates a lot of slowdown because the playback doesn't
-happen in parallel with other work. Solve this by adding a new accessibility
-thread that is in charge of speaking the document. (You don't want to use
-the compositor thread, because then that thread will become slow also.) To
-achieve this you will need to copy the accessibility tree from one thread
-to another.
-
-* *Highlighting elements during read*: The method to read the document works,
-but it'd be nice to also highlight the elements being read as it happens,
-in a similar way to how we did it for mouse hover. Implement that.
-
-* *`:hover` pseudoclass*: There is a pseudoclass for generic mouse
-[hover][hover-pseudo] events (it's unrelated to accessibility). Implement it
-by sending along mouse hover events to the active `Tab` and hit testing
-to find out which element is hovered. Try to do so by avoiding a [forced
-layout][forced-layout-hit-test]; one way to do that is to store a
-`pending_hover` on the `Tab` and running the hit test at the right time during
-`render` (which will be after layout), and then doing *another* render to
+*`:hover` pseudoclass*: There is a `:hover` pseudoclass that
```

Comment:
```suggestion
*`:hover` pseudoclass*: There is a `:hover` pseudo-class that
```

Reply:
Fixed, and fixed all other instances of pseudoclass to be pseudo-class